lib/data/mock/lost_animal_mock.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:miauche/data/interfaces/abstract_api.dart';
+import 'package:miauche/domain/models/animal_model.dart';
+
+class LostAnimalMock extends AnimalAbstractApi {
+  static List<Map> list = [
+    {
+      "animal": "Cachorro",
+      "anotherPlace": "Perto da área verde",
```

Comment:
Esse anotherPlaca significa que o animal foi perdido em algum lugar que não seja o seu. Então pra isso existe a outra variável chamada anotherPlaceAdress que vai cuidar dessa verificação. Se ela for sim, significa que esse campo deve existir e não ser nulo; se ela for não, como é o caso, essa variável fica em branco. Decida aí qual modificar, afinal é somente uma classe mock.